docs/src/user_docs/stdlib/math/u64.md

```diff
@@ -10,60 +10,43 @@ All procedures assume that an unsigned 64-bit integer (u64) is encoded using two
 [a_hi, a_lo, ... ]
 ```
 
-Procedures which check whether the input values are encoded correctly are designated with `checked` prefix. For example, `checked_add` would fail if any of the top 4 elements on the stack contains a value greater than $2^{32} - 1$. In contrast, `wrapping_add` and `overflowing_add` would not perform these checks, and therefore, if any of the top 4 stack elements is greater than $2^{32} - 1$, the operation will not fail but rather will produce an undefined result. Thus, when using versions of procedures which are not checked, it is important to be certain that input values are 32-bit limbs encoding valid u64 values.
+It is important to be certain that procedures designated with `wrapping` and `overflowing` prefixes would not perform the check whether the input values are 32-bit limbs encoding valid u64 values. For example, `wrapping_add` and `overflowing_add` would not perform these checks, and therefore, if any of the top 4 stack elements is greater than $2^{32} - 1$, the operation will not fail but rather will produce an undefined result.
```

Comment:
I think the wording here is somewhat confusing and slightly inaccurate (e.g., `lt` procedure also does not check its inputs but it doesn't have either of the two mentioned prefixes). I'd probably phrase this like so:

Many of the procedures listed below (e.g., `overflowing_add`, `wrapping_add`, `lt`) do not check whether the inputs are encoded using valid `u32` values. These procedures do not fail when the inputs are encoded incorrectly, but rather produce undefined results. Thus, it is important to be certain that limbs of input values are valid `u32` values prior to calling such procedures.